src/routes/resources/[resourceContentId]/TranslationSelector.svelte

```diff
@@ -4,7 +4,7 @@
 
     export let allLanguages: Language[];
     export let existingTranslations: ContentTranslation[];
-    export let selectedLanguageId: string | null;
+    export let selectedLanguageId: number | null;
```

Comment:
For some reason I was thinking that the `<select` would always bind value to a string. Does this work as expected since you removed the `parseInt` in the other file?

Reply:
yes. Since we define `export let selectedLanguageId: number | null;` Svelte is intelligent to make sure that `selectedLanguageId` remains a number as necessary.